www/front_src/src/Resources/Filter/index.test.tsx

```diff
@@ -46,16 +46,16 @@ import {
   searchableFields,
 } from '../testUtils';
 import useDetails from '../Details/useDetails';
+import { defaultSortOrder, defaultSortField } from '../Listing/columns';
```

Comment:
Is it better to make a global configuration variable and import it for future development and not forget a file?
Because I see that it is used several times in different files.